src/istio/ambient/chart/templates/exemptions.yaml

```
@@ -0,0 +1,39 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: uds.dev/v1alpha1
+kind: Exemption
+metadata:
+ name: istio
+ namespace: uds-policy-exemptions
+spec:
+ exemptions:
+   - policies:
+       - DisallowPrivileged # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RequireNonRootUser # CNI plugin requires root
+       - RestrictVolumeTypes # CNI plugin uses 'hostPath' volume type
+       - DisallowHostNamespaces # CNI plugin requires host namespace access for networking
+       - DropAllCapabilities # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RestrictCapabilities # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
+       - RestrictHostPathWrite # CNI plugin requires access to write to CNI
+       - RestrictHostPorts # CNI plugin requires access to host ports
```

Reviewing the pod security context I'm not sure all of these are required?
```yaml
    securityContext:
      appArmorProfile:
        type: Unconfined
      capabilities:
        add:
        - NET_ADMIN
        - NET_RAW
        - SYS_PTRACE
        - SYS_ADMIN
        - DAC_OVERRIDE
        drop:
        - ALL
      privileged: false
      runAsGroup: 0
      runAsNonRoot: false
      runAsUser: 0
```

In particular I was able to make these changes:
```suggestion
       - RequireNonRootUser # CNI plugin requires root
       - RestrictVolumeTypes # CNI plugin uses 'hostPath' volume type
       - DisallowHostNamespaces # CNI plugin requires host namespace access for networking
       - RestrictCapabilities # CNI plugin requires NET_ADMIN, NET_RAW, SYS_PTRACE, SYS_ADMIN, and DAC_OVERRIDE capabilities
       - RestrictHostPathWrite # CNI plugin requires access to write to CNI
       - RestrictHostPorts # CNI plugin requires access to host ports
```
since privileged is explicitly set as `false` and we do actually drop all capabilities (although we also add some).